plugin/src/main/scala/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -8,10 +8,15 @@ import nsc.plugins.{Plugin, PluginComponent}
 import scala.reflect.internal.util.NoPosition
 import scala.collection.mutable
 
-private[plugin] case class ChiselPluginArguments(val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty) {
+private[plugin] case class ChiselPluginArguments(
+  val skipFiles:                mutable.HashSet[String] = mutable.HashSet.empty,
+  var buildElementsAccessor:    Boolean = true,
```

Comment:
```suggestion
  var buildElementsAccessor:    Boolean = false,
```
If the option is to enable it, then it needs to be false by default.

Reply:
Yeah, I was just leaving this until the end.

Reply:
done